src/engine/ruleset/decoders/wazuh-agent/upgrade/upgrade.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: decoder/upgrade/0
+
+metadata:
+    module: wazuh-agent/upgrade
+    title: Upgrade Confirmation events
+    description: >
+        Decodes Upgrade confirmation messages
+    compatibility: >
+        This decoder has been tested on Wazuh version 4.4.0
+    versions:
+      - "4.4.0"
+    author:
+        name: Wazuh, Inc.
+        url: https://wazuh.com
+        date: 2023/03/22
+    references:
+        - https://documentation.wazuh.com/current/user-manual/agents/remote-upgrading/upgrading-agent.html#upgrading-agent
+        - https://documentation.wazuh.com/current/user-manual/agents/remote-upgrading/agent-upgrade-module.html
+
+sources:
+  - decoder/queue-upgrade/0
+
+check:
+  - event.original: +starts_with/{
+  - agent.id: +exists
```

Comment:
Is this necessary?

Reply:
agent.id check can be removed but checking the start of the event string can avoid trying to parse it as json when no tneccesary.